client/src/components/assessments/PeriodicAssessmentResults.js

```diff
@@ -144,7 +144,11 @@ export const PeriodicAssessmentsRows = ({
     assessmentConfig,
     assessmentYupSchema
   } = entity.getPeriodicAssessmentDetails(recurrence)
-  if (_isEmpty(assessmentConfig)) {
+  const filteredAssessmentConfig = Model.filterAssessmentConfig(
+    assessmentConfig,
+    entity
```

Comment:
There is no TODO here, is relatedObject not needed here?

Reply:
`relatedObject` is only for instant assessments; for periodic assessments there is no ~spoon~`relatedObject`.